src/tests/pages/App.test.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import App from '../../App';
-import { render } from '@testing-library/react-native';
+import { render, fireEvent, within } from '@testing-library/react-native';
+import { FlatList } from 'react-native';
+import '@testing-library/jest-native';
```

Comment:
need to import this to be able to use `toBeOnTheScreen` and `toHaveTextContent`. While we could check for whether the queries are null or query by text, I thought that it would also be nice if others would like to use the extended expect matchers from this library.

It's also used in an example here: https://testing-library.com/docs/react-native-testing-library/example-intro

Reply:
I do like `toBeOnTheScreen` and `toHaveTextContent` better as they are more readable than checking for whether they're null or not. :D